x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/sequence/SequenceMatcher.java

```diff
@@ -72,11 +72,13 @@ public void clear() {
     // Set of completed sequences - separate to avoid polluting the other stages
     // It is a set since matches are ordered at insertion time based on the ordinal of the first entry
     private final Set<Sequence> completed;
+    private final Set<Sequence> toCheckForMissing;
```

Comment:
I think this one can be considered for clearing up in `clear()` method as well, no?